src/lib/optimizer/strategy/between_composition_rule.cpp

```diff
@@ -360,7 +360,11 @@ BetweenCompositionRule::ColumnBoundary BetweenCompositionRule::_get_boundary(
       default:
         break;
     }
-    return {left_column_expression, value_expression, type, false, expression_id};
+    return {.column_expression = left_column_expression,
+            .border_expression = value_expression,
+            .type = type,
+            .boundary_is_column_expression = false,
+            .id = expression_id};
```

Comment:
Could we just add a constructor to such structs?

Reply:
I mean, it's nice that it is very explicit. However, it's also quite bloaty because it is so explicit. The value of this syntax for small structs, probably for two elements of different types is probably debatable. For larger structs, it gets quite long, especially for many calls. I don't really have an opinion, yet. Roundtable candidate

Reply:
Added to guidelines. Can you check the comment? I don't have an idea for a "crisp" example.